akita/src/persistState.ts

```diff
@@ -147,86 +147,88 @@ export function persistState(params?: Partial<PersistStateParams>) {
     });
   }
 
-  // when we use the local/session storage we perform the serialize, otherwise we let the passed storage implementation to do it
```

Comment:
Just `return` if there is no storage. See here https://github.com/tuurbo/akita/blob/master/akita/src/persistState.ts#L112

Reply:
ok thanks